.github/workflows/auth-e2e-tests.yml

```diff
@@ -44,13 +44,18 @@ jobs:
         with:
           project: apps/authentication-service
           start: npm run dev --workspace apps/authentication-service
-          wait-on: "http://localhost:${{ vars.PORT }}/ping"
+          wait-on: "http://localhost:8089/ping"
```

Comment:
Wes and I talked about this. There's no need to have PORT in an .env file, since only UCP will be using this.